common/components/card-component.ts

```diff
@@ -112,6 +112,10 @@ export class CardComponent<CardType extends Card = Card> {
 			this.game.otherPlayerEntity(this.slot?.player.entity),
 		)
 	}
+	/** Get the type of card */
+	public get type(): CardType {
+		return this.props
+	}
```

Comment:
Im confused, what is the purpose of this?

Reply:
That's the one thing I wasnt sure how to do so I wrote a method for it, I'm using that in the ladder test to check what hermit is in which slot. I couldnt find anything existing for getting the type of card.

Reply:
- you should be able to do `expect(CardComponent.props).toBe(TheCardObject)` I think